stockAndPrices/services/app/games.py

```diff
@@ -0,0 +1,125 @@
+from flask import Flask, Blueprint, request
+import requests, json
+
+games_blueprint = Blueprint("games", __name__, url_prefix="/games")
+api_url = "https://www.cheapshark.com/api/1.0"
+
+
+
+#### Games ####
+
+# /title?name=somename
+@games_blueprint.route('/title')
+def gamePriceByTitle():
+    game_title = validation(request.args.get('name', ''))
+    url = api_url + f"/games?title={game_title}"
+    payload, files, headers = {}, {}, {}
+    response = requests.request("GET", url, headers=headers, data=payload, files=files)
+    if response.status_code != 200:
+        return f"Get request from {url} failed.", 500
+    resp = json.loads(response.text)
+    all_ids = ''
+    all_prices = {}
+    count = 0
+    # retrieve all gameids from searching that game
+    for rs in resp:
+        if not all_ids:
+            all_ids += rs['gameID']
+        else:
+            all_ids += ',' + rs['gameID']
+        count += 1
+        if count == 25:
+            findGamePriceById(all_ids, all_prices)
+            count, all_ids = 0, ''
+
+    if not all_ids and not all_prices:
+        return "No Result"
+    elif all_ids:
+        findGamePriceById(all_ids, all_prices)
+    return all_prices
+
+def validation(title):
+    if not title:
+        return ""
+    
+    if not title.isalnum():
+        ret = ''
```

Comment:
What is the intent for creating this string, if its not alphanumeric, shouldn't we return "" or some error?

Reply:
if the title included some other symbol such as ":,[]" , I think we want to ignore them, but not the whole string? We only want the alphanumeric character inside the string.